commands/evm-call.js

```diff
@@ -0,0 +1,55 @@
+const exitOnError = require('../utils/exit-on-error');
+const web3 = require('web3');
+const { NearProvider, utils } = require('near-web3-provider');
+const assert = require('assert');
+
+module.exports = {
+    command: 'evm-call <evmAccount> <contractName> <methodName> [args]',
+    desc: 'Schedule call inside EVM machine',
+    builder: (yargs) => yargs
+        .option('gas', {
+            desc: 'Max amount of gas this call can use',
```

Comment:
That's NEAR gas. Need to be explicit, because people will confuse EVM gas with near gas

Reply:
pushed